third_party/rules_jmh/defs.bzl

```diff
@@ -0,0 +1,23 @@
+def jmh_java_benchmarks(name, srcs, deps=[], tags=[], plugins=[], main_class="org.openjdk.jmh.Main", **kwargs):
```

Comment:
Within Google's code base, we have a rule called java_jmh_benchmarks which supports name, srcs, deps, and tags (though apparently not plugins yet). Could you use that name, to make it easier for us to converge the two sets of rules on a single open-source one?